renderdoc/replay/replay_driver.h

```diff
@@ -37,6 +37,26 @@ struct FetchFrameRecord
   rdctype::array<FetchDrawcall> drawcallList;
 };
 
+struct GetTextureDataParams
+{
+  bool forDiskSave;
+  FormatComponentType typeHint;
+  bool resolve;
+  bool forceRGBA8unorm;
```

Comment:
Done. Obviously Ive only tested with REMAP_RGBA8 for now. We also have REMAP_R8 for single channel EAC.

At the moment Im just using the remap enum as a boolean, as REMAP_RGBA8 is the only path that is finished.